src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -151,28 +236,27 @@ AmendmentTableImpl::get (std::string const& name)
     return uint256 ();
 }
 
-AmendmentState*
-AmendmentTableImpl::addKnown (const char* amendmentID, const char* friendlyName,
-    bool veto)
+bool
+AmendmentTableImpl::addKnown (uint256 const& hash,
+                              std::string const& friendlyName,
+                              bool veto)
 {
-    uint256 hash;
-    hash.SetHex (amendmentID);
-
     if (hash.isZero ())
     {
         assert (false);
-        return nullptr;
+        return false;
```

Comment:
But honestly, this is a quibble.  I don't think this will ever happen, and I don't think the boolean will ever be used and I don't care so much either!